packages/rn-tester-e2e/test/features/buttonComponentScreen.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Button component screen
```

Comment:
this file should not be needed since we don't have cucumber in this PR, right?

Reply:
sure, thanks. fixed